Makefile

```diff
@@ -85,18 +86,32 @@ $(LUCENE_ANALYZER_JAR):
 
 jars: $(INDEXER_JAR) $(TAGSOUP_JAR) $(LUCENE_ANALYZER_JAR) $(LUCENE_CORE_JAR)
 
-docs/index.html: $(SOURCES)
-	mkdir -p docs
+webhelp/index.html: $(SOURCES)
```

Comment:
I don't mind changing it if you have a good reason (and are willing to do the changes in Jenkins :p) I just think I might be missing the point.